app/models/broadcast_event/library_complete.rb

```diff
@@ -37,5 +41,5 @@ def order
   has_metadata(:order_type) { |_, e| e.order.order_role.try(:role) || 'UNKNOWN' }
   has_metadata(:submission_template) { |_, e| e.order.template_name }
 
-  has_metadata(:team) { |tube, _e| tube.team || 'UNKNOWN' }
+  has_metadata(:team) { |_, e| RequestType.find(e.order.request_types.try(:first))&.product_line&.name || 'UNKNOWN' }
```

Comment:
Why it needs RequestType.find()? and not e.order.request_types.try(:first) directly? Maybe it needs reload?

Reply:
Ah because the request_types field on Order is just an array of ids - it's not a proper Rails relationship (unless I'm misunderstanding it)